Gemfile.lock

```diff
@@ -69,4 +70,4 @@ DEPENDENCIES
   jekyll-paginate
 
 BUNDLED WITH
-   2.2.32
+   2.2.24
```

Comment:
I think I could make this (and perhaps the previous) error go away if my bundler were 2.2.32.  If this causes any problems, we'll revert, and I'll figure out how to upgrade.